src/commands/release/list.ts

```diff
@@ -84,6 +89,22 @@ export default class ReleaseListCmd extends Command {
 		);
 
 		if (options.json) {
+			// We fetch the bC explicit read fields independently, since we want to fetch all fields on both bC & oB
+			// but bC & oB have different fields, so we can't enumerate them in a single $select.
+			const releasesWithExplicitReadFields =
+				await balena.models.release.getAllByApplication(slug, {
+					$select: ['id', ...explicitReadFields],
+				});
+			const _ = await import('lodash');
+			const releasesWithExplicitReadFieldsById = _.keyBy(
+				releasesWithExplicitReadFields,
+				(r) => r.id,
+			);
+
+			for (const release of releases) {
+				Object.assign(release, releasesWithExplicitReadFieldsById[release.id]);
+			}
```

Comment:
@thgreasi WDYT of this v:
```suggestion
            const mergedReleases = releases.map((release) => ({
            	...release,
            	...releasesWithExplicitReadFieldsById[release.id],
            }));
```

And then ofc use mergedReleases from there on, obviously